packaging/windows/clisdk/registrykeys.wxs

```diff
@@ -3,23 +3,9 @@
   <?include "Variables.wxi" ?>
   <Fragment>
     <ComponentGroup Id="AuthoredRegistryKeys">
-      <!-- Include the 64-bit registry keys only in the 64-bit installer -->
-      <?if $(var.Platform) = x64?>
-      <Component Id="SetupRegistry_x64" Directory="TARGETDIR" Win64="yes">
-        <RegistryKey Root="HKLM" Key="SOFTWARE\dotnet\Setup">
-          <RegistryValue Action="write" Name="Install" Type="integer" Value="1" KeyPath="yes"/>
-          <RegistryValue Action="write" Name="InstallDir" Type="string" Value="[DOTNETHOME]" />
-          <RegistryValue Action="write" Name="Version" Type="string" Value="$(var.Dotnet_ProductVersion)" />
-        </RegistryKey>
-      </Component>
-      <?endif?>
-
-      <!-- Always install the 32-bit registry keys and env vars -->
       <Component Id="SetupRegistry_x86" Directory="TARGETDIR" Win64="no">
-        <RegistryKey Root="HKLM" Key="SOFTWARE\dotnet\Setup">
-          <RegistryValue Action="write" Name="Install" Type="integer" Value="1" KeyPath="yes"/>
-          <RegistryValue Action="write" Name="InstallDir" Type="string" Value="[DOTNETHOME]" />
-          <RegistryValue Action="write" Name="Version" Type="string" Value="$(var.Dotnet_ProductVersion)" />
+        <RegistryKey Root="HKLM" Key="SOFTWARE\dotnet\Setup\InstalledVersions\$(var.Platform)\sdk">
```

Comment:
The guids are automatically generated based on the registry value that is set, the documentation is [here](http://wixtoolset.org/documentation/manual/v3/xsd/wix/component.html). And same guid is generated for the same registry value. So we should not be worrying about it.